.cypress/integration/VisualizationCharts/10_horizontalBar_chart.spec.js

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
What is the reason to rename/renumber the files?

Reply:
Was trying to follow the formatting of going in order of the visualizations appearing. After removing the visualizations we don't have I just numbered them in the order they appear in explorer.